src/main/java/edu/cornell/kfs/fp/document/service/impl/CuDisbursementVoucherExtractionHelperServiceImpl.java

```diff
@@ -530,5 +532,30 @@ public void setPaymentMethodGeneralLedgerPendingEntryService(
 	       // Split the chopped words into individual segments.
 	       return builder.toString().split("\\n");
 	   }
-
+	   
+	   @Override
+	   public boolean shouldExtractPayment(DisbursementVoucherDocument paymentSource) {
+	       LOG.debug("paymentSource: " + paymentSource.getClass());
+	       if (isRecurringDV(paymentSource)) {
+	           LOG.debug("shouldExtractPayment: found a recurring DV, returning false for " + paymentSource.getDocumentNumber());
+	           return false;
+	       }
+	       boolean shouldExtract = super.shouldExtractPayment(paymentSource);
+	       LOG.debug("shouldExtractPayment: Found a non recurring DV with a document number of " + paymentSource.getDocumentNumber() + " and returning " + shouldExtract);
+	       return shouldExtract;
+	   }
+	   
+	   private boolean isRecurringDV(DisbursementVoucherDocument paymentSource) {
+	       boolean isRecurringDV = false;
+	       try {
+    	       if (CuFPConstants.RecurringDisbursementVoucherDocumentConstants.RECURRING_DV_DOCUMENT_TYPE_NAME.equalsIgnoreCase(
+                       paymentSource.getDocumentHeader().getWorkflowDocument().getDocumentTypeName())) {
+                   isRecurringDV = true;
+               }
+	       } catch (Exception e) {
+	           LOG.error("isRecurringDV: There was a problem checking to see if the DV was a recurring DV. ", e);
```

Comment:
What exception is being thrown/caught here; can we be more explicit than just Exception? Shouldn't this wrap the exception and rethrow it as a RuntimeException? If there is a problem, we don't want to keep going do we?